eras/conway/impl/src/Cardano/Ledger/Conway/Governance.hs

```diff
@@ -176,7 +176,7 @@ instance EraPParams era => FromCBOR (ConwayGovState era) where
 data EnactState era = EnactState
   { ensCommittee :: !(StrictMaybe (Set (KeyHash 'Voting (EraCrypto era)), Rational))
   -- ^ Constitutional Committee
-  , ensConstitution :: !(SafeHash (EraCrypto era) ByteString)
+  , ensConstitution :: !(SafeHash (EraCrypto era) ByteString, StrictMaybe (ScriptHash (EraCrypto era)))
```

Comment:
I'd like it more if it was a separate field in the `EnactState`.

Reply:
Noted!

Reply:
Either that or we create a datatype called `Constitution` which holds both.

Reply:
Yes, let's create a separate `Constitution` type.